packages/frontend-v2/package.json

```diff
@@ -9,7 +9,11 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/react": "^1.8.8",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
     "@graduate/common": "workspace:*",
+    "framer-motion": "^6",
```

Comment:
why do we need framer?

Reply:
peer dependency again, perhaps used by chakra modals and things.

Link to the installation i followed: https://chakra-ui.com/guides/getting-started/nextjs-guide